File: AD-HOC/2297/main.py
test = 1
while r := int(input()):
    print(f'Teste {test}')
    total_a, total_b = 0, 0
    for _ in range(r):
        a, b = map(int, input().split())
        total_a += a
        total_b += b
    print('Aldo' if total_a > total_b else 'Beto', end='\n\n')
    test += 1

File: AD-HOC/2297/main.dart
import 'dart:io';

void main() {
  for (int test = 1, r = int.parse(stdin.readLineSync()!); r > 0; test++, r = int.parse(stdin.readLineSync()!)) {
    print('Teste $test');
    var [totalA, totalB] = [0, 0];
    while (r-- > 0) {
      var [a, b] = stdin.readLineSync()!.split(' ').map(int.parse).toList();
      totalA += a;
      totalB += b;
    }
    print(totalA > totalB ? 'Aldo\n' : 'Beto\n');
  }
}
